.travis.yml

```diff
@@ -8,11 +8,15 @@ node_js:
 - v6
 - node
 sudo: false
+
+cache:
+  directories:
+  - node_modules
+
 deploy:
   provider: npm
   email: hi@oncletom.io
-  api_key:
-    secure: UdhueW/+siuza1tueKlmLfyCxeqXB2Si53mDjz56ZXnW2rDdmBZuzHYE7IFn1u2Le2v+Eu+grGF7v9iGgEuIxMIa0JR+3Q55mZnMKHl+29fkaydq1hJgJH9wIXRu+1wistlnviK+N6fM5zt/H5taVXVV0yO/aAGgOwaFVqb1yrk=
+  api_key: "$NPM_TOKEN"
```

Comment:
Where is the $NPM_TOKEN stored and passed? In travis configuration somewhere?

Reply:
@PavelVanecek yes exactly, instead of being encrypted in the configuration file itself.